LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Ipmi.cs

```diff
@@ -218,17 +218,21 @@ private unsafe void Update(StringBuilder stringBuilder)
             }
         }
 
-        for (int i = 0; i < Math.Min(_temperatures.Count, Temperatures.Length); ++i)
-            Temperatures[i] = _temperatures[i];
+        if (Temperatures != null)
+            for (int i = 0; i < Math.Min(_temperatures.Count, Temperatures.Length); ++i)
```

Comment:
Add braces below if statement please.

Reply:
Sure, added some braces.  Let me know if the style I picked up isn't what you are looking for.

Reply:
Thanks, this is what I meant.